src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -725,7 +725,7 @@ public override void Add(View view)
         /// <since_tizen> 8 </since_tizen>
         public override void Remove(View view)
         {
-            if (SnapToPage && CurrentPage == Children.IndexOf(view) && CurrentPage == Children.Count - 1)
+            if (SnapToPage && CurrentPage == Children.IndexOf(view) && CurrentPage == Children.Count - 1 && Children.Count > 1)
```

Comment:
Actually, ScrollToIndex check the input index is positive or not.
So I think it is enough remove CurrentPage = Math.Max(0, CurrentPage - 1); at the next line.
And use ScrollToIndex(CurrentPage - 1);, because CurrentPage is also set in the ScrollToIndex as a Index value.

It is just my preference that to do not want to use specific number in the code. At this case "1" for comparing.

Reply:
Ok, I've fixed it. I think it is simple way to fix.